### DISCUSSION TITLE
Touch dragging of editor groups rejected on Android Chrome

### DISCUSSION THREAD

**Discussion by Lucent** [community/community#3601] [Codespaces]
Using a Galaxy Tab S7+ with Android 11 and Codespaces either in a web browser (Chrome 89) or as a PWA, using touches to drag editors between editor groups or split them off is rejected.

Using a Bluetooth mouse, I have no problem splitting off and moving editors, but it appears to be impossible using touch and hold/drag on screen or even click and hold using a trackpad or pen input.

**Comment by 2percentsilk** [comment]
Hey @Lucent,

Thanks for the feedback - while Codespaces works with tablets, it's definitely a work in progress as we're not currently touch optimized.
We recommend using the Bluetooth mouse and keyboard if possible but we're working on improving this experience!
